src/baguetteBox.js

```diff
@@ -148,50 +146,75 @@
         supports.svg = testSVGSupport();
 
         buildOverlay();
+        destroyGalleriesForSelector(selector);
```

Comment:
Why do we destroy on run() ? I'm not sure I understand the function's responsibility.